deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py

```diff
@@ -0,0 +1,217 @@
+import argparse
+import json
+import pandas as pd
+
+from datetime import datetime
+from os import makedirs
+from os.path import exists
+from subprocess import run
+
+# Global Configuration Variables
+csv_types = ['aggregate', 'intermediate']
+url_format = 'https://'
+
+def format_directory(directory):
+    """Check the paths of the directories and format them if necessary.
+    Args:
+        args (string): Path of a directory.
+    Returns:
+        string: Formatted path.
+    """
+    return directory if directory.endswith("/") else f'{directory}/'
+
+def check_csv_types(types):
+    """Check that the CSV type is valid.
+    Args:
+        types (list): List of types.
+    """
+    for value in types:
+        if (value not in csv_types):
+            raise Exception("CSV type is not valid") 
```

Comment:
Consider including the invalid CSV type to the exception to improve debugging

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5629/commits/ca020ab5fd076f3ff705605233b10f1ca0921806